chartmogul/__init__.py

```diff
@@ -30,7 +31,7 @@
 """
 
 __title__ = 'chartmogul'
-__version__ = '1.5.0'
+__version__ = '1.5.1'
```

Comment:
[suggestion]
```suggestion
__version__ = '1.6.0'
```
>MINOR version when you add functionality in a backwards compatible manner

[Semver](https://semver.org/)

Reply:
Done!